class/Zarieth; The Witch

```diff
@@ -0,0 +1,1836 @@
+{
+	"_meta": 
+	{
+		"sources": 
+		[
+			{
+				"json": "ZariethTheWitchFromGMBinder",
+				"abbreviation": "Brew",
+				"full": "The Witch",
+				"authors": 
+				[
+					"u/Zarieth"
+				],
+				"version": "1.1",
+				"url": "https://www.gmbinder.com/share/-L7fK9yKzbw1PUOM3iJi",
+				"targetSchema": "1.0.0"
+			}
+		],
+		"dateAdded": 1549127108
+	},
+	"class": 
+	[
+		{
+			"name": "The Witch",
+			"source": "The Witch",
+			"fluff": 
+			[	
+				{
+					"name": "",
```

Comment:
You can (and should!) remove blank `name` strings

Reply:
Done. An Artifact of figuring out why the first line of my fluff wasn't properly showing. Removed it.